src/app/IdeaPost.js

```diff
@@ -14,13 +14,12 @@ const IdeaText = styled.p`
   font-size: 0.8em;
 `
 
-export default function IdeaPost({ title, text }) {
+export default function IdeaPost({ title, text, tags }) {
   return (
     <IdeaContainer>
       <IdeaTitle>{title}</IdeaTitle>
       <IdeaText>{text}</IdeaText>
-      <Tag category="sales" />
-      <Tag category="logistics" />
+      {tags && tags.map(tag => <Tag key={tag} tag={tag} />)}
```

Comment:
If there are two tags with the same content, your key could be a problem. You could consider to use another key like the index.

Reply:
according to this article, using the thing you are mapping is a better solution. I'm guessing, because even if the characters are the same, you are referencing a different key in an object in a different array... ([Medium - Why using an index as Key in React is probably a bad idea](https://bit.ly/2K7dHfi))